synapse/datamodel.py

```diff
@@ -603,12 +616,15 @@ def addDataModels(self, mods):
 
         # load all the types in order...
         for _, mdef in mods:
+            custom = mdef.get('custom', False)
             for typename, (basename, typeopts), typeinfo in mdef.get('types', ()):
+                typeinfo['custom'] = custom
                 self.addType(typename, basename, typeopts, typeinfo)
 
         # Load all the universal properties
         for _, mdef in mods:
             for univname, typedef, univinfo in mdef.get('univs', ()):
+                # univinfo['custom'] = custom
```

Comment:
was this meant to be commented out?

Reply:
probably not - i'll need to rerun things with that. I had a bit of really noisy warning code at one point related to universal properties.